vocode/turn_based/synthesizer/google_synthesizer.py

```diff
@@ -0,0 +1,61 @@
+import io
+import os
+from typing import Optional
+from pydub import AudioSegment
+import requests
+from vocode import getenv
+from vocode.turn_based.synthesizer.base_synthesizer import BaseSynthesizer
+from google.cloud import texttospeech_v1beta1 as tts
+
+DEFAULT_GOOGLE_LANGUAGE_CODE = "en-US"
+DEFAULT_GOOGLE_VOICE_NAME = "en-US-Neural2-I"
+DEFAULT_GOOGLE_PITCH = 0
+DEFAULT_GOOGLE_SPEAKING_RATE = 1.2
+
+class GoogleSynthesizer(BaseSynthesizer):
+    def __init__(
+        self, 
+        language_code: str = DEFAULT_GOOGLE_LANGUAGE_CODE,
+        voice_name: str = DEFAULT_GOOGLE_VOICE_NAME,
+        pitch: int = DEFAULT_GOOGLE_PITCH,
+        speaking_rate: int = DEFAULT_GOOGLE_SPEAKING_RATE,
+        ):
+        self.tts = tts
+        
+        # Instantiates a client
+        credentials_path = getenv("GOOGLE_APPLICATION_CREDENTIALS")
+        if not credentials_path:
+            raise Exception(
+                "Please set GOOGLE_APPLICATION_CREDENTIALS environment variable"
+            )
+        os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = credentials_path
```

Comment:
q: why are we setting it here – shouldn't it be set already (with the checks above)?